### ISSUE TITLE
Point raffles

### ISSUE THREAD

**Issue by bttfw** [Tkd-Alex/Twitch-Channel-Points-Miner-v2#226]
Is their a way to let the script monitor for !join so if for example if 20 people write !join the script will do the same so it can participate in point raffles

**Comment by RakSrinaNa** [comment]
No. This bot is here for twitch channels points and isn't intended to mind other kind of points. If you wish to do that you can automate an IRC connection through another way.

**Comment by bttfw** [comment]
Ha I thought all points on twitch are channel points my fault but I learned something thank you

**Comment by RakSrinaNa** [comment]
Twitch channels points are only the counter below the chat. And how they are awarded is described in here too (basically by watching and through predictions).
But the streamer can't award any points by itself.
So everything you see going through the chat with commands (like !join) is probably a point system of a bit (streamlabs, etc.)

**Comment by bttfw** [comment]
Yes resarched a bit its streamlabs did not know their is a diffrence but now I know thanks for that 

Also thanks for your docker version setting that up right now
